### PR TITLE
refactor(gui-v2): use message instead of notification

### PR DESCRIPTION
**Pull request by wingkwong**
## Change Summary

- ref: #3081 

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by dstala — APPROVED** [review]

**Comment by o1lab** [comment]
Cool - much needed.

**Comment by dstala** [comment]
most of the messages are in plain English; we need to re-work this, maybe post-release.

**Comment by o1lab** [comment]
Pls get them to spreadsheet
